github_connector_branch_options/models/github_repository.py

```diff
@@ -7,25 +7,57 @@
 
 _logger = logging.getLogger(__name__)
 
+_odoo_version = [
```

Comment:
github_connector is agnostic. if you want to develop "odoo" things, please put in github_connector_odoo module.